### ISSUE TITLE
Footer component update

### ISSUE THREAD

**Issue by Memo-Es** [opencollective/opencollective#3367]
:art:  See [latest design on figma](https://www.figma.com/file/9Iq3vBOmZ2EuOacXEYD5yK/DS-05-UI-Organisms?node-id=4%3A0)

## Website footer update
---

We updated the footer component based on the feedback provided by @alanna in #3321 and also to address accessibility issues and comply with the WCAG AAA standards.

@Betree  it could probably be related with #3009

<img width="1209" alt="Screen Shot on 2020-07-28 at 23-03-16" src="https://user-images.githubusercontent.com/49539954/88755351-94955b00-d126-11ea-9ca8-ba87600c2153.png">

**Comment by Betree** [comment]
@Memo-Es The language selection is missing from this design, it's a blocker as there's currently no other way to change your language

**Comment by Memo-Es** [comment]
**Update - Language selection**

<img width="613" alt="Screen Shot on 2020-07-28 at 23-00-55" src="https://user-images.githubusercontent.com/49539954/88755252-5435dd00-d126-11ea-811d-c317aff1537f.png">

@Betree Changes are applied inside the file mentioned in the description.

**Comment by Betree** [comment]
@Memo-Es thank you, think is perfect!

**Comment by flickz** [comment]
@Betree Should I work on this as a follow up of https://github.com/opencollective/opencollective/issues/3321?

**Comment by Betree** [comment]
@flickz It's open for contributions. The main parts of this update are:
- Update the fontSizes/fontWeight/color when necessary
- Implement the country select. For flags, use the UTF8 emojis (ie. https://emojipedia.org/flag-spain/). When a language is picked, we should probably mark the select as `isLoading=true` while user is being redirected.

**Comment by Betree** [comment]
:gift: :money_with_wings: A **$100** bounty was attached to this issue. Anyone submitting a Pull Request will be rewarded with $100 when the Pull Request is reviewed, accepted and merged.

:arrow_right: [More info](https://docs.opencollective.com/help/developers/bounties)

**Comment by snitin315** [comment]
Can I take this up?

**Comment by Betree** [comment]
@snitin315 Sorry, @flickz first requested to work on this (even though there was initially no bounty on it). @flickz can you confirm that you want to work on this one?

**Comment by flickz** [comment]
Yeah sure.

**Comment by Betree** [comment]
:trophy: This issue has been completed by @flickz in https://github.com/opencollective/opencollective-frontend/pull/4765 

To claim the **$100** bounty, you can either:
- Submit an invoice to https://opencollective.com/engineering/expenses/new (please mention the issue number)
- Or ask for an Open Collective gift card of the same value (to donate to other collectives)
